clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/jobs/JobRequest.java

```diff
@@ -58,4 +61,27 @@ private CommandLine createCommandLine(List<String> tokenizedCommand) {
     commandLine.addArguments(arguments, false);
     return commandLine;
   }
+
+  private String streamToString(InputStream is) {
+    try {
+      return IOUtils.toString(is, Charset.defaultCharset());
+    } catch (Exception e) {
+      return "";
+    }
+  }
+
+  @Override
+  public boolean equals(Object o) {
+    if (this == o) return true;
+    if (o == null || getClass() != o.getClass()) return false;
+    JobRequest that = (JobRequest) o;
+    return tokenizedCommand.equals(that.tokenizedCommand) &&
+      environment.equals(that.environment) &&
+      streamToString(inputStream).equals(streamToString(that.inputStream));
```

Comment:
we need to compare stream contents. otherwise the equality will fail. hence why i just didn't use lombok here